docs/sql-guide/functions/function-charindex.md

```diff
@@ -64,3 +64,15 @@ RESULTS
 | _id | charindex|
```

Comment:
Forgive me, I can't work out what this is...? Is it a table? or something else?

It's confusing using "charindex" as a column name which is identical to the SQL function because a user won't be able to determine which is the command and which is the column/value.

It doesn't matter what the column/value name actually is so long as there's a cear distinction between it and the SQL command/function/whatever :)

Reply:
This doesn't match other examples so just copying like we have for other pages